### ISSUE TITLE
[kaeld on 2008-01-08 05:13:28] : StringHub cannot handle full load of SLC DOMs

### ISSUE THREAD

**Issue by dglo** [dglo/pdaq#716]
This issue is being opened to track the various performance problems associated with processing DOMs in Soft LC (SLC) mode.  I ran a test in November at Pole which failed miserably when all DOMs on a string were configured to readout the SLC hits.  The load spiked and the run quickly crashed.

**Comment by dglo** [comment]
[kaeld on 2008-01-08 05:26:28]
Today I ran a series of quick runs on SPTS64's S-Cube system.  I placed the DOMs in pulser mode at 305 Hz and SLC (so that the cable B/W was not exceeded) and I found that Omicron which represents the front-half of the StringHub was able to handle SLC without any problem at this rate.  Note that this system did break down at 610 Hz.  Also note that the CPU on scube is 1.8 GHz instead of 1.4 GHz.
